src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -5,16 +5,69 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.util.UUID;
+import java.time.Instant;
 
 /**
  * Servlet class responsible for user registration.
  */
 public class RegisterServlet extends HttpServlet {
 
- @Override
- public void doGet(HttpServletRequest request, HttpServletResponse response)
-     throws IOException, ServletException {
+  /**
+  * Store class that gives access to Users.
```

Comment:
Another SUPER minor style-guide nit (see all the caveats in my comment below): It's standard to align javadoc and block comments on the asterisks, so this line would be indented one extra space. It's up to the team whether this is something you'd all like to follow.

Google style guide: https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line
Oracle: http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141999.html#16838

Reply:
Fixed 👍